addons/binding/pom.xml

```diff
@@ -19,5 +19,6 @@
   <modules>
     <module>org.openhab.binding.enocean</module>
     <module>org.openhab.binding.loxone</module>
+    <module>org.openhab.binding.freemobilesms</module>
```

Comment:
Do not add to the "addons" directory. As you can see that directory is currently emptied and gets removed in a few days.

Reply:
Moved to bundles.